homeassistant/components/switch/rachio.py

```diff
@@ -8,7 +8,7 @@
 from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
 from homeassistant.const import CONF_ACCESS_TOKEN
 
-REQUIREMENTS = ['rachiopy==0.1.2']
+REQUIREMENTS = ['rachiopy==0.1.1']
```

Comment:
0.1.2 was a version I created in my fork of RachioPy for A/B testing. I decided that 0.1.1 works better, so I never pushed 0.1.2 upstream.